### PR TITLE
Fixes #16003 - Adding unattended to capsule remove

### PR DESCRIPTION
**Pull request by chris1984**
adding option to do --unattended for capsule remove.

### CODE REVIEW

File: bin/capsule-remove [hunk 1]
```diff
@@ -84,17 +45,15 @@ LOG_FILES=(
     /var/log/foreman-installer
     /var/log/squid
 )
-
+
```
Comment (beav) [inline bin/capsule-remove]: it looks like some extraneous whitespace snuck in

File: bin/capsule-remove [hunk 2]
```diff
@@ -84,17 +45,15 @@ LOG_FILES=(
...
+ 
 FQDN=$( hostname -f)
-
+ 
 RPMS=(
     `rpm -qa | grep puppetlabs-release`
     `rpm -qa | grep foreman-release`
     `rpm -qa | grep foreman-client`
-    `rpm -qa | grep ^foreman-installer`
-    `rpm -qa | grep ^katello-installer`
```
Comment (beav) [inline bin/capsule-remove]: are these rpms kept on the system after uninstall?

**Comment by beav** [comment]
I had a question about `katello-installer` and `foreman-installer`, but patch looks good overall

File: bin/capsule-remove [hunk 3]
```diff
@@ -130,28 +89,89 @@ RPMS=(
...
     /etc/pki/pulp/
     /etc/pki/katello
     /root/ssl-build
     /var/www/html/pub/katello-ca*.rpm
 )
-
+ 
 # stopping all katello/foreman related services
+ 
+  echo "Stopping Satellite Capsule"			| tee -a $LOG
```
Comment (ehelms) [inline bin/capsule-remove]: This is not Satellite :)

File: bin/capsule-remove (hunk 2 above)
Comment (chris1984) [inline bin/capsule-remove]: @beav lines 58 and 66 are removing those with the # rpm -qa | grep ^foreman and ^katello so removing extra lines of code. Tested on a install and those greps return nothing after the script runs

**Comment by chris1984** [comment]
@beav Let me know if you want me to post the output of the script I have a capsule snap shotted I have been running this against.

**Comment by beav** [comment]
@chris1984 i need to test it still, but the code itself looks fine. Testing now.

File: bin/capsule-remove [hunk 4]
```diff
@@ -130,28 +87,88 @@ RPMS=(
...
-rm -rf ${CONFIG_FILES[*]}
-
-echo "Cleaning up log files"
-# logs
-rm -rf ${LOG_FILES[*]}
+# In case of --help or -h
+for var in $@; do
+   if [[ $var == "--help" || $var == "-h" ]]
+       then 
+           echo -e "This script is used to remove all capsule related packages and services. \nThis should only be used if you are sure you are wanting to completely remove Capsule from this machine and lose all of your settings and synced content."
```
Comment (beav) [inline bin/capsule-remove]: The `--help` output needs updating to mention the `--unattended` option

**Comment by beav** [comment]
script works as advertised, it is good to go after adding the help text

**Comment by chris1984** [comment]
@beav Thanks again for checking, can I get a merge if it looks good?

**Comment by beav** [comment]
@chris1984 ya, ACK from me

**Comment by chris1984** [comment]
@ehelms can I have get a merge please :)

**Comment by beav** [comment]
@chris1984 sorry, my mistake. I acked but did not hit the button.

**Pull request merged**